server/database/queries/events/get-events-category.js

```diff
@@ -2,9 +2,11 @@ const connection = require('../../connection');
 
 module.exports = async (category, date) => {
   const { rows } = await connection.query(`
-    SELECT * FROM events 
-    WHERE category = $1 AND expire_date >= $2 
-    ORDER BY attendance DESC `,
+    SELECT events.*, organization.name
+    FROM events INNER JOIN organization
+    ON events.organizer_id = organization.id  
+    WHERE events.category = $1 AND expire_date >= $2 
```

Comment:
```suggestion
    WHERE events.category LIKE ? AND expire_date >= $2 
```
You have an array of categories and you search inside it, you have to use the `LIKE`  PSQL keyword